library/ZendService/Twitter/Twitter.php

```diff
@@ -115,8 +122,19 @@ class Twitter
      * @param  null|OAuth\Consumer $consumer
      * @param  null|Http\Client $httpClient
      */
-    public function __construct($options = null, OAuth\Consumer $consumer = null, Http\Client $httpClient = null)
-    {
+    public function __construct(
+        $options = null,
+        OAuth\Consumer $consumer = null,
+        Http\Client $httpClient = null,
+        Response $response = null
+    ) {
+        if (is_null($response)) {
+            $this->twitterResponse = new Response($httpClient);
```

Comment:
A response accepts an http response as constructor parameter, not an http client